### PR TITLE
Add netcore50 configuration to System.Threading.Tasks

### PR DESCRIPTION
**Pull request by mellinoe**
@weshaggard

### CODE REVIEW

File: src/System.Threading.Tasks/src/System.Threading.Tasks.csproj [hunk 1]
```diff
@@ -6,22 +6,31 @@
     <AssemblyName>System.Threading.Tasks</AssemblyName>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <!-- Monitor type-forward seen as duplicate type -->
```
Comment (weshaggard) [inline src/System.Threading.Tasks/src/System.Threading.Tasks.csproj]: Not sure I understand that. The type-forward gets injected by genfacades correct? 

If it is coming from code we should try to scope the warning if possible with pragma's instead.

**Comment by weshaggard** [comment]
LGTM

File: src/System.Threading.Tasks/src/System.Threading.Tasks.csproj (hunk 1 above)
Comment (mellinoe) [inline src/System.Threading.Tasks/src/System.Threading.Tasks.csproj]: I'll check if I can disable it with a pragma. I just copied this over from the internal project and can't remember why I initially added it.

Comment (mellinoe) [inline src/System.Threading.Tasks/src/System.Threading.Tasks.csproj]: I don't think this is needed at all anymore.

**Pull request merged**